source/jormungandr/jormungandr/scenarios/distributed.py

```diff
@@ -103,17 +103,23 @@ def _compute_journeys(self, future_manager, request, instance, krakens_call, con
         logger.debug("request_id : {}".format(request_id))
 
         requested_dep_modes_with_pt = {
-            mode for mode, _, direct_path_type in krakens_call if direct_path_type != "only"
+            mode
+            for mode, _, direct_path_type in krakens_call
+            if direct_path_type not in ("only", "only_with_alternatives")
```

Comment:
Since you're often using a test on the type, maybe you can extract a function from it? `is_type_only` or something like this?